css/index.styl

```diff
@@ -779,6 +779,11 @@
 
     .auth0-lock-badge
       display inline-block
+      color rgba(white,.7)
+      font-size 14px
+      font-family "Avenir"
```

Comment:
that line font-family I think is not necessary we can delete and use the "Avenir Next" of the .auth0-lock.auth0-lock class.